docs/uml-class-diagrams/admin/test/DemoTests/DemoTests.puml

```diff
@@ -2,7 +2,13 @@
 
 package edu.oakland.test.admin {
     class DemoTests {
-
+        + void rfidNumberIsNotNull()
+        + void storageInputLocIsNotNull()
+        + void storageOutputLocIsNotNull()
```

Comment:
Just occurred to me that rfid is an int... and thus it can't be null. Then it occurred to me than an empty string is another edge case we need to handle. What do you think of this?
```suggestion
        + void storageInputLocIsNotNull()
        + void storageOutputLocIsNotNull()
        + void storageInputLocIsNotEmpty()
        + void storageOutputLocIsNotEmpty()
```